includes/class-convertkit-pmp-api.php

```diff
@@ -124,9 +124,23 @@ public function add_tag_to_user( $user_email, $user_name, $tag_id ) {
 			'email' => $user_email,
 		);
 
+		// If there are custom fields, add them to the body of the API request.
+		if ( ! empty( $subscribe_fields ) ) {
+			$args['fields'] = $subscribe_fields;
+		}
+
+		// If there is more than one tag, add the rest as additional tags in the body of the API request.
+		if ( count( $subscribe_tags ) > 1 ) {
+			$primary_tag_id = array_shift( $subscribe_tags );
+			$args['tags'] = $subscribe_tags;
+		} else {
+			// Set the primary tag ID for the API request to the only item in the array of $subscribe_tags.
+			$primary_tag_id = array_values( $subscribe_tags )[0];
+		}
+
 		// Build the request URL.
 		$query_args = array();
-		$request_url = $this->api_url . '/' . $this->api_version . '/tags/' . $tag_id . '/subscribe';
+		$request_url = $this->api_url . '/' . $this->api_version . '/tags/' . $primary_tag_id . '/subscribe';
```

Comment:
Should we prepare this variable in any way? Is it always a safe variable to put into the URL?

Reply:
We expect an integer so could intval it.
```suggestion
		$request_url = $this->api_url . '/' . $this->api_version . '/tags/' . intval( $primary_tag_id ) . '/subscribe';
```